src/Utilities/test/runtests.jl

```diff
@@ -5,16 +5,83 @@ using Test
 
 using Utilities.MoistThermodynamics, PlanetParameters
```

Comment:
I think that this needs to be simply `using Utilities, PlanetParameters`, so that `LinearAlgebra` is brought into scope.